### DISCUSSION TITLE
How to setup custom shortcut/key combination ( Paste from previous! )

### DISCUSSION THREAD

**Discussion by joaomfaria** [rsjaffe/MIDI2LR#588] [Q&A] [answered]
Hi,

How can i set up custom shortcut/macros to each key?
There is a module to shortcuts for user or command sequence but i have not figured out how to access and define those. 

Sorry for the noob question and thanks for your time!

**Comment by rsjaffe** [comment, marked as answer]
see the wiki: https://github.com/rsjaffe/MIDI2LR/wiki/Plugin-Options-Dialog#series-of-commands
